parameter/ConfigurableDomain.cpp

```diff
@@ -829,13 +829,25 @@ string CConfigurableDomain::getLastAppliedConfigurationName() const
 // Pending configuration
 string CConfigurableDomain::getPendingConfigurationName() const
 {
-    const CDomainConfiguration* pPendingConfiguration = getPendingConfiguration();
+    const CDomainConfiguration* pApplicableDomainConfiguration = findApplicableDomainConfiguration();
 
-    if (pPendingConfiguration) {
+    if (!pApplicableDomainConfiguration) {
 
-        return pPendingConfiguration->getName();
+        // No configuration is pending
+        return "<none>";
```

Comment:
Conflicting with commit message.
Now: if there's no applicable configuration the pending configuration will be shown as "";
Should be: if there's no applicable configuration the pending configuration will be shown as "<none>";